packages/fluentui/projects-test/README.md

```diff
@@ -1,3 +1,14 @@
-For now, all the project tests files reside in `scripts/gulp/tasks/test-projects`. This project serves as a unit of
-modularity, so builds can optimize based on package scripts. In the future, we'll be moving those tests into
-this package.
+The purpose of tests in this package is to ensure that `@fluentui/react-northstar` and its subpackages can be consumed without issues in different environments and bundlers.
```

Comment:
Now that this package isn't using gulp, it could potentially be moved under `/apps` since that's where we've traditionally put test-runner projects like this. (Or leaving it here for now is fine if you prefer.)

Reply:
For now I would prefer to keep it in `packages/fluentui` as it also uses a separate Lerna config (`packages/fluentui/lerna.json`)